two-entity-poc/src/util/error-middleware.ts

```diff
@@ -0,0 +1,11 @@
+import { Request, Response, NextFunction } from "express";
+
+interface IError extends Error{
+    statusCode: number;
+}
+
+export const errorMiddleware = (err: IError, req: Request, res: Response, next: NextFunction) => {
```

Comment:
create folder middleware, and put middleware function there

Reply:
Done